src/pages/GamePage.js

```diff
@@ -1,42 +1,65 @@
 import 'App.css';
 import 'style.css';
-import LeagueContainer from 'components/LeagueContainer';
+import LeagueCard from 'components/LeagueCard';
+import { Carousel } from '@trendyol-js/react-carousel';
 import { useParams } from 'react-router-dom';
 import { useState, useEffect } from 'react';
 import { http } from 'http';
+import SeriesContainer from 'components/SeriesContainer';
+
+function handleClick() {
+  window.location.href = '/';
+}
 
 const GamePage = () => {
   const params = useParams();
   const [gameName, setGameName] = useState(params.gameName);
   const [leagues, setLeagues] = useState([]);
+  const [resize, setResize] = useState(window.innerWidth);
 
   useEffect(async () => {
     const { leagues: response, name } = await http(
       `https://api.pandascore.co/videogames/${params.gameName}`
     );
     setGameName(name);
     setLeagues(
-      response.map(({ image_url, name }) => {
-        return { image: image_url, title: name };
+      response.map(({ image_url, name, series }) => {
+        return { image: image_url, title: name, series };
       })
     );
   }, []);
+  useEffect(() => {
+    function handleResize() {
+      setResize(window.innerWidth);
+    }
+
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+

```

Comment:
You do not need to define two different use effects. It can be in the same one you just need to return a arrow function from the first one